examples/test_frontend_socket/assets/package.json

```diff
@@ -2,6 +2,6 @@
   "name": "test_frontend",
   "dependencies": {
     "phoenix": "^1.4.0",
-    "pushex": "0.2.0"
+    "pushex": "^1.0.0"
```

Comment:
@Kwilliams-dev303 you may want to update here as well. The only change is upgrading the Phoenix library to 1.7.12 as well